src/contracts/contracts-directive.js

```diff
@@ -1,121 +1,139 @@
-angular.module('ua.contracts').directive('contracts', function () {
-    "use strict";
-    return {
-        restrict : 'EA',
-        replace  : true,
-        require  : '^ngModel',
-        templateUrl: 'components/ovh-utils-angular/contracts/contracts.html',
-        scope : {
-            contracts : '=',
-            agree     : '=ngModel'
-        },
-        link : function ($scope, $elm, $attr) {
-
-            $scope.fullText = $attr.fullText === "true" || $attr.fullText === undefined;
-
-            var scrollToOptions = {
-                'easing'  : 'swing',
-                'duration' : '300',
-                'offsetTop'  : '16'
-            };
-
-            $scope.disabled = true;
-
-
-            $scope.$watch('contracts', function (nv) {
-                if (nv !== undefined) {
-                    init();
-                }
+import template from './contracts.html';
+
+export default function () {
+  return {
+    restrict: 'EA',
+    replace: true,
+    require: '^ngModel',
+    template,
+    scope: {
+      contracts: '=',
+      agree: '=ngModel',
+    },
+    link($scope, $elm, $attr) {
```

Comment:
missing `ngInject`?

Reply:
Nop 

https://docs.angularjs.org/api/ng/service/$compile#-link-

Link is called by angular itself :)